pkg/mls/api/v1/service.go

```diff
@@ -118,45 +118,22 @@ func (s *Service) RegisterInstallation(ctx context.Context, req *mlsv1.RegisterI
 		return nil, err
 	}
 
-	if req.IsInboxIdCredential {
```

Comment:
Removed support for legacy credentials, since we no longer even have an `account_address` field to store the resulting address.